### PR TITLE
argon2: add to oi-userland

### PR DESCRIPTION
**Pull request by cgrzemba**
another prerequisite for keepassxc

### CODE REVIEW

**Comment by AndWac** [comment]
This package is a perfect example on how to add test results:
mkdir test && touch test/results-all.master && gmake test && cp build/test/amd64/results-64.snapshot test/results-all.master && gmake test
git add test/results-all.master

File: components/sysutils/argon2/Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+COMPONENT_FMRI= security/argon2
+COMPONENT_CLASSIFICATION= Applications/System Utilities
+COMPONENT_LICENSE_FILE=LICENSE
+
+include $(WS_MAKE_RULES)/common.mk
+
+COMPONENT_TEST_TARGETS= test
+
+test: $(TEST_64)
+
```
Comment (AndWac) [inline components/sysutils/argon2/Makefile]: This is the default for BUILD_BITS=64

**Comment by cgrzemba** [comment]
Pleas modify your hint:
`mkdir test && touch test/results-all.master && gmake test; cp build/test/amd64/results-64.snapshot test/results-all.master && gmake test;
git add test/results-all.master`
the first running of `gmake test` have to fail because of the empty test/results-all.master

**Comment by AndWac** [comment]
It is exactly working as intended. Without a results-all.master (results-32.master, results-64.master) the machinery won‘t start. An empty one is the simplest one to start with.

**Comment by AndWac** [comment]
Please squash the commits.

**Pull request merged**